### PR TITLE
Add Uninstall Instructions for Teleport Connect

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 1]
```diff
@@ -317,3 +317,41 @@ $ "%LocalAppData%\Programs\teleport-connect\Teleport Connect.exe" --insecure
 ````
 </TabItem>
 </Tabs>
+
+## Uninstall Teleport Connect
```
Comment (ptgott) [inline docs/pages/connect-your-client/teleport-connect.mdx:321]:
```suggestion
## Uninstalling Teleport Connect
```
To match the language of the other H2s

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 2]
```diff
@@ -317,3 +317,41 @@ $ "%LocalAppData%\Programs\teleport-connect\Teleport Connect.exe" --insecure
...
+
+<Tabs>
+<TabItem label="macOS">
+
+Remove Teleport Connect for MacOS from the Applications with this command.
```
Comment (ptgott) [inline docs/pages/connect-your-client/teleport-connect.mdx:326]:
```suggestion
Remove Teleport Connect for MacOS from the Applications directory with this command:
```

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 3]
```diff
@@ -317,3 +317,41 @@ $ "%LocalAppData%\Programs\teleport-connect\Teleport Connect.exe" --insecure
...
+
+(!docs/pages/includes/uninstall-teleport-connect-windows.mdx!)
+
+</TabItem>
+<TabItem label="Linux">
+
+For DEB installations uninstall Teleport Connect using APT:
+
+```code
+sudo apt remove teleport-connect
```
Comment (ptgott) [inline docs/pages/connect-your-client/teleport-connect.mdx:343]:
```suggestion
$ sudo apt remove teleport-connect
```

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 4]
```diff
@@ -317,3 +317,41 @@ $ "%LocalAppData%\Programs\teleport-connect\Teleport Connect.exe" --insecure
...
+```
+
+For RPM installations uninstall Teleport Connect using YUM:
+
+```code
+sudo yum remove teleport-connect
+```
+
+Installs that used the tarball download and extraction should remove the
```
Comment (ptgott) [inline docs/pages/connect-your-client/teleport-connect.mdx:352]:
```suggestion
Installs based on a tarball should remove the
```

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions. Disabled auto-merge so you can apply them.

File: docs/pages/management/admin/uninstall-teleport.mdx [hunk 5]
```diff
@@ -408,10 +404,7 @@ $ docker stop teleport
   ```code
   $ del C:\Path\To\tsh.exe
   ```
-
-  You can uninstall Teleport Connect from the "Apps and Features" section of the Control Panel.
-
-  For reference, Teleport Connect binaries are installed to `%LOCALAPPDATA%\Programs\teleport-connect`.
+(!docs/pages/includes/teleport-connect-telemetry.mdx!)
```
Comment (gzdunek) [inline docs/pages/management/admin/uninstall-teleport.mdx:407]: Telemetry? 
I think you meant `(!docs/pages/includes/uninstall-teleport-connect-windows.mdx!)`

**Review by gzdunek — COMMENTED** [review]
The uninstall-teleport.mdx page contains also the instructions how to remove the app data. WDYT about including them here too, or at least, adding a link to them?

File: docs/pages/management/admin/uninstall-teleport.mdx (hunk 5 above)
Comment (stevenGravy) [reply docs/pages/management/admin/uninstall-teleport.mdx]: Thanks, fixed!

**Comment by stevenGravy** [comment]
> The uninstall-teleport.mdx page contains also the instructions how to remove the app data. WDYT about including them here too, or at least, adding a link to them?

Yes, will add that in.

**Review by zmb3 — APPROVED** [review]

**Pull request merged**